### PR TITLE
[DC-596] Remove project field from normalized datasets

### PR DESCRIPTION
**Pull request by s-rubenstein**
This removes the project field from `normalizeDataset` (reducing the number of added fields). It also tangentially fixes a bug where if a dataset has multiple consortiums, we would only render the first.

Filters aren't quite working yet.

### CODE REVIEW

File: src/pages/library/dataBrowser-utils.js [hunk 1]
```diff
@@ -56,6 +56,12 @@ export const isDatarepoSnapshot = dataset => {
   return _.toLower(dataset['dcat:accessURL']).includes(datarepoSnapshotUrlFragment)
 }
 
+export const renderConsortium = dataset => _.flow(
+  _.map(dataCollection => dataCollection['dct:title']),
```
Comment (pshapiro4broad) [inline src/pages/library/dataBrowser-utils.js]: This seems fine, although it makes me wonder about where we want this mapping to end up once our refactor is done (or further along). My thinking is that we could have the object that contains the data know how to modify it to produce the UI ready version (so this would be a method on that object) or maybe the rendering of data is separate from the data, but still refactored so it's not duplicated.

File: src/pages/library/dataBrowser-utils.js [hunk 2]
```diff
@@ -56,6 +56,12 @@ export const isDatarepoSnapshot = dataset => {
...
+  _.join(', ')
+)(dataset['TerraDCAT_ap:hasDataCollection'])
```
Comment (pshapiro4broad) [inline src/pages/library/dataBrowser-utils.js]: Maybe this could be refactored a bit more now. It seems like there should be a way to combine this code with the code in `Databrowser.js`.

File: src/pages/library/Showcase.js [hunk 3]
```diff
@@ -112,7 +112,7 @@ const Showcase = () => {
   return h(FooterWrapper, { alwaysShow: true }, [
     libraryTopMatter('featured workspaces'),
     h(SearchAndFilterComponent, {
-      fullList, sidebarSections,
+      fullList, sidebarSections, nameField: 'name',
```
Comment (pshapiro4broad) [inline src/pages/library/Showcase.js]: Is this related to these changes? I'm not sure what this change is doing.

**Review by pshapiro4broad — APPROVED** [review]

File: src/pages/library/Showcase.js (hunk 3 above)
Comment (s-rubenstein) [reply src/pages/library/Showcase.js]: This snuck in from me starting to look at removing `lowerName`. Will revert.

File: src/pages/library/dataBrowser-utils.js (hunk 1 above)
Comment (s-rubenstein) [reply src/pages/library/dataBrowser-utils.js]: I think its a little more complicated by that. In this case, nothing in our schema/dataset knows that this is where consortium is pulled from. That actually feels like an oversight, in my book. Because there are two fields, it would be weird to me to say that it is rendered solely by pulling out `'dct:title'`

File: src/pages/library/dataBrowser-utils.js (hunk 2 above)
Comment (s-rubenstein) [reply src/pages/library/dataBrowser-utils.js]: I bounced around a few ideas in my head but the tldr is that I think, yeah, there probably is. I will look into it.

**Review by snf2ye — APPROVED** [review]
